Client/Client.Qt/ConnectionManager.hpp

```diff
@@ -79,12 +79,14 @@ class ReceiverManager : public QObject
 class ConnectionManager : public Network::Client
 {
 public:
+    ConnectionManager() { configureConnectionProperties(); }
```

Comment:
Better to keep one style. If all implementation is moved to cpp than all implementations should be moved to cpp file.